com.woltlab.wcf/templates/__multifactorTotpCodeField.tpl

```diff
@@ -3,7 +3,7 @@
 	*}name="{@$field->getPrefixedId()}" {*
 	*}value="{if !$field->isI18n() || !$field->hasI18nValues() || $availableLanguages|count === 1}{$field->getValue()}{/if}" {*
 	*}class="multifactorTotpCode" {*
-	*}autocomplete="off" {*
+	*}{if $field->getAutoComplete() !== null}autocomplete="{$field->getAutoComplete()}" {/if}{*
```

Comment:
This should probably be moved down for consistency with `__textFormField.tpl`. Applies likewise to the other MFA fields (and possibly also the `class`).